apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -93,21 +93,19 @@ function ChooseSharingNameModal({
       <button type="submit" class="btn btn-primary">Choose Sharing Name</button>
     `;
   } else {
-    body = html`
-    <strong>Unable to change your course's sharing name.</strong>
-    </p>
-    <p>
-      Your course's sharing name cannot be changed because at least one question has
-      been shared. Doing so would break the assessments of other courses that have
-      imported your questions.
-    </p>`;
+    body = html` <strong>Unable to change your course's sharing name.</strong>
+      <p>
+        Your course's sharing name cannot be changed because at least one question has been shared.
+        Doing so would break the assessments of other courses that have imported your questions.
+      </p>`;
     footer = html`
       <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
     `;
   }
   return Modal({
     title: 'Choose Sharing Name',
     id: 'chooseSharingNameModal',
+    form: canChooseSharingName,
     body,
```

Comment:
These changes seem unrelated, but they are actually necessary for the accessibility tests.

Previously, the accessibility tests never saw this version of the modal, because a sharing name wasn't initialized for the test course. However this cause the problem that the Public questions page and Public assessment pages were not accessible.

I changed the accessibility tests so that the sharing name _is_ initialized so that the public questions and public assessment pages can be accessed, and then this cause a problem I needed to fix over here.

Bigger Picture: we should probably have some way to make sure that accessibility tests are actually seeing the different versions of a page rather than just one.